src/app/checkout/page.tsx

```
@@ -0,0 +1,400 @@
+      onEvent: (event) => {
+        console.log("Checkout event:", event.event_type, event.data);
+        switch (event.event_type) {
+          case "checkout.opened":
+            // Checkout successfully opened
+            clearTimeout(loadingTimeout);
+            console.log("Checkout opened successfully");
+            break;
+
+          case "checkout.breakdown":
+            // Update order breakdown when pricing details change
+            const message = event.data?.message as CheckoutBreakdownData;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)